contracts/Core/StakeManager.sol

```diff
@@ -224,7 +225,7 @@ contract StakeManager is Initializable, ACL, StakeStorage, StateManager, Pause {
     function setCommission(uint8 commission) external {
         uint32 stakerId = stakerIds[msg.sender];
         require(stakerId != 0, "staker id = 0");
-        require(stakers[stakerId].acceptDelegation, "Delegetion not accpected");
+        require(!stakers[stakerId].acceptDelegation, "Delegation accepted");
```

Comment:
Hey @adi44 
Is this require needed anymore ?
Without this well seq is assured
As you cant accept delegation unless you have setCommission
So if you are calling setCommission, its given you have not accepted delegation.

Reply:
Yes true, we can remove that. it seems to be redundant.

Reply:
cool